cypress/support/step-definitions/action-popover-steps.js

```diff
@@ -0,0 +1,25 @@
+import { actionPopoverButton, actionPopover, actionPopoverInnerItem } from '../../locators/action-popover';
+import { eventInAction } from '../../locators';
```

Comment:
Do you use it somewhere? If not remove it, please

Reply:
fixed